src/frontend/Komponenter/Behandling/Vurdering/tekster.ts

```diff
@@ -1,3 +1,13 @@
+export const teksterMedKursiv = [
```

Comment:
I stedenfor "hva" vi gjør - kunne navngivning si noe om hvorfor? 

Gamle vilkårstekster gjelder for : "stønad før 1. september 2023"